CMakeLists.txt

```diff
@@ -127,7 +127,7 @@ else()
 endif()
 
 if (WIN32)
-  set(PKG_CONFIG_EXECUTABLE "C:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
+  set(PKG_CONFIG_EXECUTABLE "D:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
```

Comment:
If this path is specific to the CI, should we make allowing customizing it? What was this path on your local Windows machine (if you were able to try this)?

Reply:
On local machine it is C:, I think best way to handle this is to specify the location for Chocolatey installs so we can use C: for everything, unless customers are finding that it sometimes does get directed to D: for them as well. Going to look into this now.

Reply:
Okay, we can keep it D:\. Let's a note in the readme for this, mentioning that the customers can change this if this path is different in their installation

Reply:
I tried setting gstreamer installation path using by passing an install argument to choco command, and that resulted in neither the C nor D drive having the gstreamer installation. I ended up putting a check in CMake to see if the C drive path exists, else use D drive.